### PR TITLE
Update RotateAction.js

### PR DESCRIPTION
**Pull request by titiodus**
Changed var edit to let edit. I chose let over const because of the variable name edit - as this connotes changes is expected.

Hope my PR is accepted and merged.

Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Hello, thank you for your contribution, what issue is this for, can you please link to it? Are you sure someone else has not already claimed it? Thank you!

File: src/edit/actions/RotateAction.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ L.RotateAction = L.EditAction.extend({
   },
 
   addHooks() {
-    var edit = this._overlay.editing;
+    let edit = this._overlay.editing;
```
Comment (TildaDares) [inline src/edit/actions/RotateAction.js:16]:
```suggestion
    const edit = this._overlay.editing;
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @titiodus, feel free to just click the `Commit Suggestion` button. Thank you!

**Comment by titiodus** [comment]
@TildaDares I have done so. Thanks

**Review by TildaDares — APPROVED** [review]
Amazing work @titiodus. Thank you!

**Comment by TildaDares** [comment]
Congratulations on merging your PR @titiodus 🎉

**Comment by titiodus** [comment]
Thank you @TildaDares. I am willing to do more. I can coordinate contributors and such. I am alao available to handle any other issues.

**Pull request merged**